src/main/java/mate/academy/spring/model/CinemaHall.java

```diff
@@ -4,13 +4,16 @@
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.validation.constraints.NotNull;
 
 @Entity
 public class CinemaHall {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+    @NotNull
```

Comment:
Do we need to validate entity? Seems that we need to validate data that incomes from client side